rcl/test/rcl/test_security.cpp

```diff
@@ -120,16 +120,16 @@ class TestGetSecureRoot : public ::testing::Test
 
 TEST_F(TestGetSecureRoot, failureScenarios) {
   EXPECT_EQ(
-    rcl_get_secure_root(TEST_SECURITY_CONTEXT_ABSOLUTE, &allocator),
+    rcl_get_secure_root(TEST_ENCLAVE_ABSOLUTE, &allocator),
     (char *) NULL);
   rcl_reset_error();
 
   putenv_wrapper(ROS_SECURITY_ROOT_DIRECTORY_VAR_NAME "=" TEST_RESOURCES_DIRECTORY);
 
   /* Security directory is set, but there's no matching directory */
-  /// Wrong security context
+  /// Wrong enclave
   EXPECT_EQ(
-    rcl_get_secure_root("some_other_security_context", &allocator),
+    rcl_get_secure_root("some_other_enclave", &allocator),
     (char *) NULL);
   rcl_reset_error();
 }
```

Comment:
@mikaelarguedas , I was cross checking that this PR had also updated the recent changes from https://github.com/ros2/rcl/pull/609 , but I'm not seeing the added `successScenarios_local_root_security_context` test in this PR.

Reply:
Good point :+1:

The test was part of this branch as it had been rebased but was not updated.
The test did fail on the rebased job https://ci.ros2.org/job/ci_windows/9974/testReport/junit/rcl/TestGetSecureRoot/successScenarios_local_root_security_context/ and should be fixed by https://github.com/ros2/rcl/pull/612/commits/f0d86fc9cfd65257686f55e62b94c1706e0c3eec